xml/System.Linq/Enumerable.xml

```diff
@@ -9902,7 +9902,7 @@ Only unique elements are returned.
   
  If `source` contains fewer than `count` elements, an empty <xref:System.Collections.Generic.IEnumerable%601> is returned. If `count` is less than or equal to zero, all elements of `source` are yielded.  
   
- The <xref:System.Linq.Enumerable.Take%2A> and <xref:System.Linq.Enumerable.Skip%2A> methods are functional complements. Given a sequence `coll` and an integer `n`, concatenating the results of `coll.Take(n)` and `coll.Skip(n)` yields the same sequence as `coll`.  
+ The <xref:System.Linq.Enumerable.Take%2A> and <xref:System.Linq.Enumerable.Skip%2A> methods are functional complements. Given an unrepeatable sequence `coll` and an integer `n`, concatenating the results of `coll.Take(n)` and `coll.Skip(n)` yields the same sequence as `coll`.  
```

Comment:
I'm not sure I understand the meaning of the phrase "unrepeatable sequence". Could you clarify?

Reply:
An unrepeating sequence; one who's items only are iterated over once in the iteration of the sequence as a whole.

Reply:
> one who's items only are iterated over once in the iteration of the sequence as a whole.

I see your point, but I think the purpose of this comment is to provide an illustration on the semantics of `Take` and `Skip`, rather than state a universal invariant. I don't think that the particular change adds clarity to the statement. How about something like "Given a collection sequence"?

Reply:
That sounds better. I'm not able to edit this any more since I deleted the fork a week ago. Could you add in this change?

Reply:
You can't update this without the original fork. I've cloned this PR and made the change. Would you @steveberdy and @eiriktsarpalis like me to open a new PR with the change?

Reply:
I don't mind. Just mention this PR in your pull request so I can close it when yours gets merged.

Reply:
@adegeo I'll submit another PR for GC.xml. I'm assuming your PR just has changes to Enumerable.xml?

Reply:
I'll clone this PR so it will have all the changes. Is that OK?

Reply:
Yes